lib/uplinkc/bucket-ext.go

```diff
@@ -75,6 +77,8 @@ func UploadObject(cBucket C.BucketRef_t, path *C.char, reader *File, cOpts *C.Up
 	}
 }
 
+<<<<<<< HEAD:lib/uplinkc/bucket-ext.go
```

Comment:
Aren't this lines causing problems?

Reply:
Note this has been closed and no, since this was work in progress and file was ignored.